docs/configuration.html

```diff
@@ -20,13 +20,22 @@
 
   <h3 class="anchor-heading"><a id="brokerconfigs" class="anchor-link"></a><a href="#brokerconfigs">3.1 Broker Configs</a></h3>
 
-  The essential configurations are the following:
+  For ZooKeeper clusters, brokers must have the following configuration:
   <ul>
-      <li><code>broker.id</code>
-      <li><code>log.dirs</code>
-      <li><code>zookeeper.connect</code>
+    <li><code>broker.id</code></li>
+    <li><code>log.dirs</code></li>
+    <li><code>zookeeper.connect</code></li>
   </ul>
 
+  For KRaft clusters, brokers and controllers must have the following configurations:
+  <ul>
+    <li><code>node.id</code></li>
+    <li><code>log.dirs</code></li>
+    <li><code>process.roles</code></li>
+  </ul>
+
+  On KRaft brokers, if <code>broker.id</code> is set, it must be equal to <code>node.id</code>.
```

Comment:
Is broker.id documented on this page (I couldn't find it)?  If not, should this be a hyperlink to that page, or is that no necessary?

Reply:
The docs for the individual configs are included via `  <!--#include virtual="generated/kafka_config.html" -->`. We dynamically generate those from KafkaConfig.scala